assets/scss/templates/_signup.scss

```diff
@@ -74,13 +74,20 @@
 
   .feature-panel {
     flex: 4;
-    height: 100vh;
     padding: $dp-spaces--lv5 $dp-spaces--lv10;
 
+    .content-img{
+      margin: $dp-spaces--lv0;
```

Comment:
Hi! only for understand. `$dp-spaces--lv0` the lv and the number is means there a list of margins are paddings?

Reply:
Si fede! creamos esas variables para unificar el criterio y tratar de hablar el mismo idoma con diseño. Se crearon los niveles de espacios que saltan de 6px en 6px.